src/taskpane/server.js

```diff
@@ -3455,6 +4219,33 @@ function getFromSpiraExcel(model, fieldTypeEnums) {
   })
 }
 
+// resets validations - used before a GET command
+// @param: sheetName - current sheet name
+function resetSheet(model) {
+
+  Excel.run(function (ctx) {
+    var fields = model.fields;
+    //complete data range from old data
+    var sheet = ctx.workbook.worksheets.getActiveWorksheet();
+    var range = sheet.getRangeByIndexes(1, 0, EXCEL_MAX_ROWS, fields.length);
+    range.delete(Excel.DeleteShiftDirection.up);
+
+    ctx.sync();
+
+    var dataBaseSheetName = params.dataSheetName + model.currentProject.id + model.currentArtifact.id;
+
+    //clear database worksheet
+    var worksheet = context.workbook.worksheets.getItemOrNullObject(dataBaseSheetName);
+    worksheet.getRange().clear();

```

Comment:
In this case do you think we should clear the sheet or delete it completely? What is the value in keeping it?

Reply:
As far as I remember, there was a bug on Excel API for deleting worksheets, sometimes triggering exceptions. So, to be safe, I decided to just delete a range of rows and reuse the same sheet.